ComprefaceTestApp/Services/ExampleSubjectService.cs

```diff
@@ -51,4 +58,59 @@ public async Task<ListAllExampleSubjectResponse> GetAllExampleSubjects(ListAllEx
 
         return response;
     }
+
+    public async Task<DeleteAllExamplesResponse?> ClearSubjectAsync(DeleteAllExamplesRequest request)
+    {
+        var requestUrl = $"{_httpClient.BaseAddress}recognition/faces";
+
+        var response = await requestUrl.
+            SetQueryParam("subject", request.Subject)
+            .DeleteAsync(HttpCompletionOption.ResponseContentRead);
+
+        return await response.ResponseMessage.Content.ReadFromJsonAsync<DeleteAllExamplesResponse>();
+    }
+
+    public async Task<DeleteImageByIdResponse?> DeleteImageByIdAsync(DeleteImageByIdRequest request)
+    {
+        var requestUrl = $"{_httpClient.BaseAddress}recognition/faces";
+
+        var response = await requestUrl
+            .AppendPathSegment(request.ImageId.ToString())
+            .DeleteAsync(HttpCompletionOption.ResponseContentRead);
+
+        return await response.ResponseMessage.Content.ReadFromJsonAsync<DeleteImageByIdResponse>();
+    }
+
+    public async Task<List<Face>> DeletMultipleExamples(DeleteMultipleExamplesRequest deleteMultipleExamplesRequest)
```

Comment:
for return type it is better to create seperate response class for it here because this `List<Face>` does not fit with the api response contract, in the response there are other fields other than `faces` field. You can see the full list of response fields in the fiddler

Reply:
yeah, I was doing this way, but it threw an exception.
![image](https://user-images.githubusercontent.com/39197664/209154215-c3160155-e9be-461d-9cc2-0eecf6ab760a.png)